symplyphysics/laws/hydro/pressure_of_liquid_in_vessel_moving_horizontally.py

```diff
@@ -0,0 +1,64 @@
+from sympy import (Eq, solve, sqrt)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+from sympy.physics.units import acceleration_due_to_gravity as earth_free_fall_acceleration
+
+from symplyphysics.laws.hydro import hydrostatic_pressure_from_density_and_depth_acceleration as pressure_law
+
+# Description
+## If a vessel with a liquid moves with horizontal acceleration, then the hydrostatic pressure of the liquid
+## depends on the density of the liquid, the acceleration of free fall, the horizontal acceleration of the vessel
+## and the height.
```

Comment:
What is the height here?
For example, assume we have zero gravity, and some horizontal acceleration. How to calculate height in this case?